frontend/src/console/components/CoursesManage/displayMessages.ts

```diff
@@ -0,0 +1,35 @@
+const messages = {
+  title: {
+    defaultMessage: 'Courses (Studio)',
+    description: 'The component title'
+  },
+  paragraph_01: {
+    defaultMessage:
+      "Courses are created and managed in Studio. Here's what you need to know:",
+    description: 'First paragraph'
+  },
+  step_01: {
+    defaultMessage:
+      'Select "manage courses" below to access Studio. You\'ll be navigated to a new website.',
+    description: 'First step explanation'
+  },
+  step_02: {
+    defaultMessage:
+      'No need to sign up! Log into Studio using the same email address and password you use on OpenCraft.',
+    description: 'Second step explanation'
+  },
+  button_text: {
+    defaultMessage: 'Manage courses',
+    description: 'Button text'
+  },
+  help_text: {
+    defaultMessage: 'Need help? Visit',
```

Comment:
`Need help?` is one string and `Visit support` is another one.

Reply:
Applied, although this doesn't completely match the mockup. Please confirm.